libexec/git-elegant-deliver-work

```diff
@@ -9,5 +9,9 @@ default() {
     fi
     boxtee git fetch
     boxtee git rebase ${RMASTER}
-    boxtee git push --set-upstream --force origin ${BRANCH}:${BRANCH}
```

Comment:
@alexbeatnik you have to check if this function accepts an argument, then it has to be a name of the remote branch.
Like
```bash
local REMOTE_BRANCH=${BRANCH}
if [ -n "${1}" ]; then
    REMOTE_BRANCH=${1}
fi
boxtee git push --set-upstream --force origin ${BRANCH}:${REMOTE_BRANCH}
```

And please add a new unit test for verification this logic.